php/libraries/Database.class.inc

```diff
@@ -974,9 +974,11 @@ class Database
         if ($params) {
             $query = str_replace(array_keys($params), array_values($params), $query);
         }
-        print "$this->_databaseName:"
-            . date("j-M-Y G:i:s", time())
-            .": $query<br>\n";
+        $query = json_encode($query);
+        print "console.info('"
```

Comment:
It seems to print out the output as a string and append it to body for some reason
![image](https://cloud.githubusercontent.com/assets/6627543/20849797/4b412b88-b8a6-11e6-8925-dd92b9fbd83b.png)

This breaks the page and code never gets to execute your part of code